utils/paste.tsx

```diff
@@ -71,13 +87,30 @@ export function formatMarkdownTableMessage(table: HTMLTableElement, message?: st
     }).join('\n');
 
     const formattedTable = `${header}${body}\n`;
+    return formattedTable;
+}
+
+export function formatMarkdownMessage(clipboardData: DataTransfer, message?: string, caretPosition?: number): string {
+    const html = clipboardData.getData('text/html');
+
+    const service = new TurndownService({emDelimiter: '*'});
```

Comment:
we should probably create a service from a file that has all of our settings baked in already. This is out of scope for this PR, but could you add a line with a `TODO@michel` in it? I need to create an instance of it anyways for the WYSIWYG editor